xbmc/windowing/osx/WinSystemOSX.mm

```diff
@@ -791,10 +791,20 @@ static void DisplayReconfigured(CGDirectDisplayID display,
   if (!view)
     return false;
 
-  // if we are not starting up windowed, then hide the initial SDL window
-  // so we do not see it flash before the fade-out and switch to fullscreen.
-  if (CDisplaySettings::GetInstance().GetCurrentResolution() != RES_WINDOW)
+  if (CDisplaySettings::GetInstance().GetCurrentResolution() == RES_WINDOW)
+  {
+    // it seems, that in macOS 10.15 this defaults to YES, but we currently do not support
+    // Retina resolutions properly. Ensure that the view created by SDL uses a 1 pixel per
+    // point framebuffer.
+    if ([view respondsToSelector:@selector(setWantsBestResolutionOpenGLSurface:)])
```

Comment:
there's no need for the `-respondsToSelector:` check - it always evaluates to `YES`, as `wantsBestResolutionOpenGLSurface` property is available since macOS 10.7 and Kodi's deployment target is 10.9.

Replace these 2 lines to just `view.wantsBestResolutionOpenGLSurface = NO;`

Reply:
Thanks, will do.